docs/getting_started/singularity.md

```diff
@@ -0,0 +1,109 @@
+# Running Labelmerge with Singularity
+
+## Pre-requisites
+1. Singularity / Apptainer is is installed on your system. For more info, see
+the detailed [Apptainer install instructions](https://apptainer.org/docs/admin/main/installation.html#install-from-pre-built-packages).
+1. The following command-line tools are installed:
+    * wget
+1. Sufficient disk-space (rough estimate)
```

Comment:
Can soften these requirements a lot (probably just get rid of this list item)

Reply:
This comment still stands -- I don't think people need anything like 16GB memory to run this